internal/webhooks/cluster.go

```diff
@@ -916,21 +916,22 @@ func (webhook *Cluster) validateClusterClassExistsAndIsReconciled(ctx context.Co
 		case apierrors.IsNotFound(clusterClassPollErr):
 			allWarnings = append(allWarnings,
 				fmt.Sprintf(
```

Comment:
Did a bit of manual testing and realized that whe warnings should be improved a bit